bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/TelescopePosTableWidget.java

```diff
@@ -701,18 +724,21 @@ private void _resetTable(final TargetEnvironment env) {
      */
     void updatePrimaryStar() {
         if (_env == null || !OTOptions.isEditable(_obsComp.getProgram(), _obsComp.getContextObservation())) return;
-        final SPTarget target = getSelectedPos();
-        if (target != null) {
-            PrimaryTargetToggle.instance.toggle(_dataObject, target);
+
+        final Option<SPTarget> targetOpt = getSelectedPos();
+        if (targetOpt.isDefined()) {
+            final boolean autoGroup = targetOpt.flatMap(this::getTargetGroup).exists(igg -> igg.group().isAutomatic());
+            if (!autoGroup)
+                PrimaryTargetToggle.instance.toggle(_dataObject, targetOpt.getValue());
         } else {
-            final Option<Tuple2<Integer, GuideGroup>> groupOpt = getSelectedGroup();
-            groupOpt.foreach(group -> {
+            final Option<IndexedGuideGroup> iggOpt = getSelectedGroup();
+            iggOpt.foreach(igg -> {
                 final TargetEnvironment env = _dataObject.getTargetEnvironment();
                 final GuideGroup primary    = env.getOrCreatePrimaryGuideGroup();
-                if (primary != group._2() && confirmGroupChange(primary, group._2())) {
+                if (primary != igg.group() && confirmGroupChange(primary, igg.group())) {
```

Comment:
Let's make a note to test this too.  You have to set up custom guiding for offset positions.